task.py

```diff
@@ -342,6 +347,19 @@ def get_pod_ip(self) -> str:
         r = self.run_oc(f"get pod {self.pod_name} -o yaml", die_on_error=True)
         y = yaml.safe_load(r.out)
         return typing.cast(str, y["status"]["podIP"])
+    
+    def get_secondary_ip(self) -> str:
+        jsonpath = '{.metadata.annotations.k8s\.ovn\.org\/pod-networks}'
+        logger.info(f"get pod {self.pod_name} -o jsonpath='{jsonpath}'")        
```

Comment:
not sure about this logging line. Note that `self.run_oc()` already does DEBUG logging with more information (in particular, the result of the command). 

At INFO level, it would be better to just log the secondary IP that was received at the end of this function, instead of "we are about to request some information from the pod, but don't log the actual result".